src/main/resources/annotations.yaml

```diff
@@ -83,7 +87,10 @@ genomes:
 
   hs37-1kg:
     species: Homo sapiens
-    alias: b37
+    ucsc_alias: hg19
+    names:
+      - hs37-1kg
```

Comment:
not consistent with hg19 and some other annotations: do we need to add `build` name to names or not? In some sections it is included (this one) in other (e.g. hg19) not included.

P.S: I think we don't need duplication here.

Reply:
That's because `hg19` is not the build's release name, it's the UCSC alias, whereas `hs37-1kg` is a release name, so everything is fairly consistent here from my point of view.

The `build` property is a denotation of the assembly internal to our project, we can call them whatever we want (and since there is no universal naming scheme for genome assemblies, we don't have a better option).